src/cf.data.pre

```diff
@@ -4995,10 +4995,33 @@ DOC_START
 		[http::]<pt	Peer response time in milliseconds. The timer starts
 				when the last request byte is sent to the next hop
 				and stops when the last response byte is received.
-		[http::]<tt	Total time in milliseconds. The timer
-				starts with the first connect request (or write I/O)
-				sent to the first selected peer. The timer stops
-				with the last I/O with the last peer.
+		[http::]<tt	Total peering time in milliseconds.
+
+				The timer starts when Squid decides to forward the request (to
+				an origin server or cache_peer) and peer selection begins. The
+				timer stops when relevant forwarding activities (including any
+				retries) end.
+
+				Between those two timer events, Squid may perform DNS lookups,
+				query external ACL helpers, adapt responses using pre-cache
+				RESPMOD services, and participate in other concurrent
+				secondary activities. Most secondary activities increase
+				peering time. In some cases, a secondary activity may start
+				before the timer starts or end after the timer stops, leading
+				to misleading results of simple computations like %<tt - %dt.
```

Comment:
A secondary activity may start before the transaction peering timer when a transaction is joining an already pending secondary activity. For example, an HTTP request handled by FwdState (`%<tt` timer) may join an already ongoing DNS lookup for the requested domain name (`%dt` timer).

Similarly, a secondary activity may continue beyond a peering transaction phase. For example, a DNS A lookup may still be pending while the transaction proceeds with the DNS AAAA lookup result received earlier (due to Happy Eyeballs optimizations).

These natural imperfect/unpredictable concurrent activity/transaction overlaps imply that `%<tt - %dt` math does not always work as more "primitive" sequential processing models may suggest. In some cases, it may even yield negative results. We cannot really "fix" that, but we can make `%<tt` much simpler by not trying to exclude the "initial" DNS lookups from it (like the official code does, leading to admin confusion and Squid bugs). This PR adjusts `%<tt` definition and code accordingly.